components/bt/host/bluedroid/stack/include/stack/bt_types.h

```diff
@@ -24,15 +24,9 @@
 #include "bt_common.h"
 #include "osi/semaphore.h"
 
-typedef uint8_t UINT8;
```

Comment:
@PSONALl  Why these defines are removed from bluedroid stack include files?

Reply:
@Sushant-Espressif It was removed from here and added to "components/bt/common/include/bt_common.h" so that it can be used in case of NimBLE as well in this example